.solcover.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  istanbulReporter: ["html", "lcov"],
+  istanbulReporter: ["html", "lcov", "text", "json", "cobertura"],
```

Comment:
If we want to keep this clean, we can just report the covertura xml file. I don't think we are using the other formats